server/utils/cloudFunctions/functions/algoliaCloudFunctions/prodAlgoliaFunctions.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Description: Cloud functions for PROD firebase->algolia features. 
+ *              Cloud functions must be created as static JS objects
+ *              hence not everything can be DRY. 
+ *              Three basic operations supported: create, update, delete.
+ *              Vocab: a firestore collection = algolia index
+ *                     an ad entity = algolia record
```

Comment:
To keep consistent with Firestore vocab, use "ad document" as opposed to "ad entity"